Firestore/core/test/firebase/firestore/model/document_set_test.cc

```diff
@@ -41,78 +42,74 @@ class DocumentSetTest : public testing::Test {
   Document doc1_ = Doc("docs/1", 0, Map("sort", 2));
   Document doc2_ = Doc("docs/2", 0, Map("sort", 3));
   Document doc3_ = Doc("docs/3", 0, Map("sort", 1));
-
-  DocumentSet DocSet(initializer_list<Document> docs) const {
-    return testutil::DocSet(comp_, docs);
-  }
 };
```

Comment:
Yes. This change did not actually compile. I thought it was trivial and sent it to travis but it turns out not to work.

The problem is that some usages of the full form still exist. With these changes in place those overloads could not be resolved. Rather than make this even more complicated I just decided to revert.